codebuild_specs/sh-files/aws-configure-credentials.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
Is there a reason this script is written in python?

Reply:
This was converted from aws_configure.exp (expect) to support usage with aws_session_token / short-lived credentials; python happens to be pretty convenient here @goldbez can this be converted to js or was there another reason behind it? was it issues with ts-node?

Reply:
adding this to [backlog](https://app.asana.com/0/1203848575872011/1204779956749780/f)